pkg/cmd/job/view/view.go

```diff
@@ -124,23 +122,27 @@ func runView(opts *ViewOptions) error {
 	}
 
 	opts.IO.StartProgressIndicator()
-
 	job, err := getJob(client, repo, jobID)
 	if err != nil {
 		return fmt.Errorf("failed to get job: %w", err)
 	}
 
-	opts.IO.StopProgressIndicator()
-
 	if opts.Log {
-		opts.IO.StartProgressIndicator()
 		r, err := client.JobLog(repo, jobID)
 		if err != nil {
 			return err
 		}
+
 		opts.IO.StopProgressIndicator()
 
-		if _, err := io.Copy(out, r); err != nil {
+		opts.IO.DetectTerminalTheme()
```

Comment:
If we are not going to render markdown, then `DetectTerminalTheme()` isn't useful— it would just output some terminal escape sequences to stdout, but the result would never get used.

Reply:
that was fully cargo coded, thanks